### DISCUSSION TITLE
Dead and unconscious characters often glitch around platforms in multiplayer

### DISCUSSION THREAD

**Discussion by Regalis11** [FakeFishGames/Barotrauma#14730] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Dead and unconscious characters often start to glitch when they're on platforms, jumping up and down as if they were attached to the platform with a rubber band. 

I think this happens because the ragdolls go through platforms when any of their limbs is below it, and client-side the positional correction logic isn't accurate enough to ensure it can move all of the character's limbs above the platform. 

### Reproduction steps

1. Play any game mode in multiplayer
2. Kill some characters around platforms and stairs, you should eventually see some get stuck in this glitchy state.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, closing discussion.
